nursery/create-restart-manager-session.yml

```diff
@@ -0,0 +1,9 @@
+rule:
+  meta:
+    name: create Restart Manager session
```

Comment:
It may be useful to add a description and/or reference to how this is used in malware. I think most recently a lot of ransomware has been using this, e.g. see
- https://techdator.net/revil-ransomware-2-2-version-uses-windows-restart-manager-to-encrypt-more-files/ 
- https://www.carbonblack.com/blog/tau-threat-discovery-conti-ransomware/
- https://www.binarydefense.com/threat_watch/sodinokibi-ransomware-now-encrypts-more-files/

Reply:
You are right! I saw this used in a ransomware sample to free up files locked by a running application. I'll get some documentation added to the rule.